### PR TITLE
Add instructions for changing the user

### PR DESCRIPTION
**Pull request by larsbergstrom**
r? @aneeshusa

---
This change is [Reviewable](https://reviewable.io/reviews/servo/saltfs/741)

### CODE REVIEW

File: docs/salt.md [hunk 1]
```diff
@@ -80,6 +80,8 @@ Note that your prompt will change once you're inside a VM.
 
 * `vagrant@vagrant-ubuntu-trusty-64$ sudo apt-get -y install cowsay; cowsay 'servo-saltfs ❤  Vagrant'`:
   Run commands in the VM
+* `vagrant@vagrant-ubuntu-trusty-64$ sudo su - servo`:
+  Change to user Servo
```
Comment (aneeshusa) [inline docs/salt.md]: nit: Servo -> servo

**Comment by aneeshusa** [comment]
r=me with the nit fixed.
Friendly reminder to [please not @-mention in commit messages or PR descriptions](https://groups.google.com/forum/#!topic/mozilla.dev.servo/9jUSv0ZQQE0) :)

**Comment by larsbergstrom** [comment]
Sure, will do - I thought that was just for servo due to the weird mirroring process gecko has!

**Comment by larsbergstrom** [comment]
Updated - r? @aneeshusa

**Comment by jdm** [comment]
Yeah, the username thing only matters for servo/servo PRs.

**Pull request merged**